sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-servicebus-messaging/README.md

```diff
@@ -0,0 +1,80 @@
+# Spring Cloud Azure Service Bus Messaging Sample shared library for Java
```

Comment:
Let's change this module's name to azure-spring-cloud-sample-messaging-servicebus, and for the other one, change it to xxx-messaging-eventhubs.

Reply:
all right